articles/cms/wordpress/extending.md

```diff
@@ -209,17 +248,124 @@ CSS is retrieved from the wp-admin settings page.
-add_filter( 'auth0_login_css', 'a0_docs_ex_auth0_login_css' );
+add_filter( 'auth0_login_css', 'auth0_theme_hook_auth0_login_css' );
+```
+
+### auth0_login_form_tpl
+
+Filters the template used for the Auth0 login form. This should return a path to a file containing HTML that replaces what is in `wp-content/plugins/auth0/templates/auth0-login-form.php`. The standard Lock initiation JS looks for an ID attribute of `auth0-login-form` to instantiate the login form so make sure that's present or replace the `wp-content/plugins/auth0/assets/js/lock-init.js` file with your own. 
+
+```
+/**
+ * Override the Lock login form template.
+ *
+ * @param string  $tpl_path - original template path.
+ * @param array   $lock_options - Lock options.
+ * @param boolean $show_legacy_login - Should the template include a link to the standard WP login?
+ *
+ * @return string
+ */
+function auth0_theme_hook_auth0_login_form_tpl( $tpl_path, $lock_options, $show_legacy_login ) {
+	return AUTH0_THEME_ROOT . '/templates/auth0-login-form.html';
+}
+add_filter( 'auth0_login_form_tpl', 'auth0_theme_hook_auth0_login_form_tpl', 10, 3 );
+```
+
+### auth0_settings_fields
+
+This filter is used to modify an existing form field or to add a new one. This should return a modified `$options` array with your changes or additions. New fields must have a field callback, as shown below. 
+
+```
+/**
+ * Modify existing or add new settings fields.
+ *
+ * @param array  $options - array of options for a specific settings tab.
+ * @param string $id      - settings tab id.
+ *
+ * @return array
+ *
+ * @see WP_Auth0_Admin_Generic::init_option_section()
+ */
+function auth0_theme_hook_auth0_settings_fields( $options, $id ) {
+	switch ( $id ) {
+		case 'basic':
+			$options[] = array(
+				'name'     => __( 'A Custom Basic Setting' ),
+				'opt'      => 'custom_basic_opt_name',
+				'id'       => 'wpa0_custom_basic_opt_name',
+				'function' => 'auth0_theme_render_custom_basic_opt_name',
+			);
+			break;
+		case 'features':
+			break;
+		case 'appearance':
+			break;
+		case 'advanced':
+			break;
+	}
+	return $options;
+}
+add_filter( 'auth0_settings_fields', 'auth0_theme_hook_auth0_settings_fields', 10, 2 );
+
+/**
+ * Callback for add_settings_field
+ *
+ * @param array $args - 'label_for' = id attr, 'opt_name' = option name
+ *
+ * @see auth0_theme_hook_auth0_settings_fields()
+ */
+function auth0_theme_render_custom_basic_opt_name( $args ) {
+	$options = WP_Auth0_Options::Instance();
+	printf(
+		'<input type="text" name="%s[%s]" id="%s" value="%s">',
+		esc_attr( $options->get_options_name() ),
+		esc_attr( $args['opt_name'] ),
+		esc_attr( $args['label_for'] ),
+		esc_attr( $options->get( $args['opt_name'] ) )
+	);
+}
+```
+
+### auth0_auth_scope
+
+This filter allows developers to add or change the scope requested during login. This can be used to add [custom claims](https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims#custom-claims) or request a refresh token. 
```

Comment:
make the URL relative: `/api-auth/tutorials/adoption/scope-custom-claims#custom-claims`